### PR TITLE
Integrations to packs - Phase 8 (Dan)

### PR DESCRIPTION
**Pull request by dantavori**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Moved the following from Legacy to packs:
- integration-ArcSightXML.yml
- integration-Autofocus.yml
- integration-AWS_Sagemaker.yml
- integration-AWS-CloudWatchLogs.yml
- integration-AWS-GuardDuty.yml
- integration-AWS-Route53.yml
- integration-AWS-SecurityHub.yml
- integration-AWS-SQS.yml
- integration-BigFix.yml
- integration-Blockade.io.yml

### CODE REVIEW

File: Packs/AWS-CloudWatchLogs/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "IT Services"
+    ],
+    "tags": [
+        "AWS - CloudWatchLogs"
```
Comment (yuvalbenshalom) [inline Packs/AWS-CloudWatchLogs/pack_metadata.json:13]: remove

File: Packs/AWS-GuardDuty/pack_metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "IT Services"
+    ],
+    "tags": [
+        "AWS - GuardDuty"
```
Comment (yuvalbenshalom) [inline Packs/AWS-GuardDuty/pack_metadata.json:13]: remove

File: Packs/AWS-Route53/pack_metadata.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "IT Services"
+    ],
+    "tags": [
+        "AWS - Route53"
```
Comment (yuvalbenshalom) [inline Packs/AWS-Route53/pack_metadata.json:13]: remove

File: Packs/AWS-SQS/TestPlaybooks/playbook-AWS-SQS-Test.yml [hunk 4]
```diff
@@ -1,31 +1,26 @@
-id: fd93f620-9a2d-4fb6-85d1-151a6a72e46d
+id: AWS - SQS Test Playbook
```
Comment (yuvalbenshalom) [inline Packs/AWS-SQS/TestPlaybooks/playbook-AWS-SQS-Test.yml:1]: you should change all other occurrences in the repo.
specifically the conf.json

File: Packs/AWS-SQS/pack_metadata.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "IT Services"
+    ],
+    "tags": [
+        "AWS - SQS"
```
Comment (yuvalbenshalom) [inline Packs/AWS-SQS/pack_metadata.json:13]: remove

File: Packs/AWS-SecurityHub/pack_metadata.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "IT Services"
+    ],
+    "tags": [
+        "AWS - Security Hub"
```
Comment (yuvalbenshalom) [inline Packs/AWS-SecurityHub/pack_metadata.json:13]: remove

File: Packs/AWS_Sagemaker/pack_metadata.json [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "AWS Sagemaker"
```
Comment (yuvalbenshalom) [inline Packs/AWS_Sagemaker/pack_metadata.json:13]: remove

File: Packs/ArcSightXML/pack_metadata.json [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Analytics & SIEM"
+    ],
+    "tags": [
+        "ArcSight XML"
```
Comment (yuvalbenshalom) [inline Packs/ArcSightXML/pack_metadata.json:13]: remove

File: Packs/AutoFocus/Integrations/integration-Autofocus.yml [hunk 9]
```diff
@@ -554,4 +554,4 @@ script:
     description: Check file reputation of the given hash
   runonce: false
 tests:
-- autofocus_test
```
Comment (yuvalbenshalom) [inline Packs/AutoFocus/Integrations/integration-Autofocus.yml:557]: why remove

File: Packs/BigFix/pack_metadata.json [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Vulnerability Management"
+    ],
+    "tags": [
+        "BigFix"
```
Comment (yuvalbenshalom) [inline Packs/BigFix/pack_metadata.json:13]: remove

File: Packs/Blockade.io/pack_metadata.json [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "Blockade.io"
```
Comment (yuvalbenshalom) [inline Packs/Blockade.io/pack_metadata.json:13]: remove

File: Packs/AutoFocus/Integrations/integration-Autofocus.yml (hunk 9 above)
Comment (dantavori) [reply Packs/AutoFocus/Integrations/integration-Autofocus.yml:557]: because this test doesn't exist

**Comment by yuvalbenshalom** [comment]
seems we are bumping to the sdk problem with AWS-Route53.yml that is failing the create content artefacts.
lets remove this integration from the branch.

**Comment by yuvalbenshalom** [comment]
@dantavori check conflicts

**Pull request merged**